README.md

```diff
@@ -5,17 +5,27 @@ Synchronizes a github repository with a local repository. Automatically deals wi
 
 # Installation
 
-If you don't use `pipsi`, you're missing out.
-Here are [installation instructions](https://github.com/mitsuhiko/pipsi#readme).
+Before installing, make sure you have a **Python 3** installation with
+the Jupyter environment installed.
 
-Simply run:
+To install gitautosync, first install the package with `pip`:
 
-    $ pipsi install .
+    pip install gitautosync
+
+Next, enable the server extension with the following command:
+
+    jupyter serverextension enable --py nbgitautosync --sys-prefix
 
 
 # Usage
 
-To use it:
+To use gitautosync, open the notebook. Replace `tree` (and everything)
+after it with the following syntax:
 
-    $ gitautosync --help
+    myjupyterhub.com/git-sync?repo=<URL-to-my-repo>
```

Comment:
if you're using this with jupyterhub, it should actually be:

myjupyterhub.com/hub/user-redirect/git-sync?repo=<URL-to-your-repo>

if you're testing locally it should just be

localhost:8888/git-sync?repo=<url-to-my-repo>